plugins/services/src/js/utils/FrameworkUtil.js

```diff
@@ -76,6 +84,62 @@ const FrameworkUtil = {
     }
 
     return splitVersion.slice(1).join("-");
+  },
+
+  /**
+   * Return a warning or an error label
+   * if the pacakge has cosmosPackage been updated soon.
+   * @param {object} cosmosPackage The cosmos package to check.
+   * @param {Boolean} withLabel Indicates whether we want the "Last Updated: " label or not.
+   * @returns {null | String | React.ReactNode} Null or string or an HTML saying when was the last update of this package
+   */
+  getLastUpdated(cosmosPackage, withLabel) {
+    // This looks like magic, but we are only decoding the package configuration into a string,
+    // and then finding the PACKAGE_BUILD_TIME_STR property.
+    const v2AppMustacheTemplate = Util.findNestedPropertyInObject(
+      cosmosPackage,
+      "marathon.v2AppMustacheTemplate"
+    );
+    if (!v2AppMustacheTemplate) {
+      return null;
+    }
+
+    const decodedBuildTime = atob(v2AppMustacheTemplate).match(
+      /PACKAGE_BUILD_TIME_STR": "(.*)"/
+    );
+
+    if (!decodedBuildTime) {
+      return null;
+    }
+
+    const packageUpdatedTime = decodedBuildTime[1];
+    const now = Date.now();
+
+    const difference = now - DateUtil.strToMs(packageUpdatedTime);
+
+    let className = "";
+    if (difference > YEAR) {
+      className = "-danger";
+    } else if (difference > SIX_MONTHS) {
+      className = "-warning";
+    }
+
+    // Format to only show the date.
+    const formattedDate = DateUtil.msToUTCDay(packageUpdatedTime);
+
+    if (!withLabel) {
+      return (
+        <span className={"last-updated-package" + className}>
```

Comment:
You could have avoided writing styles by using the [Text component](https://dcos-labs.github.io/ui-kit/?path=/story/style-utilities-typography-text--color)
```
let dateColor;
 if (difference > YEAR) {
      dateColor = red;
 else if (difference > SIX_MONTHS) {
      dateColor = yellow;
}
<Text color={dateColor} dataCy="last-updated-package">
```

This is definitely not a blocker, just sharing info :)